great_expectations/rule_based_profiler/expectation_configuration_builder/expectation_configuration_builder.py

```diff
@@ -44,6 +59,11 @@ def __init__(
 
         self._expectation_type = expectation_type
 
+        self._validation_parameter_builders = init_rule_parameter_builders(
+            parameter_builder_configs=validation_parameter_builder_configs,
+            data_context=self._data_context,
+        )
```

Comment:
Could `init_rule_parameter_builders` live on a class?

Reply:
@cdkini Please see this comment/plan: https://github.com/great-expectations/great_expectations/pull/4524#issuecomment-1076637493 -- thanks!

Reply:
Thanks for closing the loop on this @alexsherstinsky. I actually didn't know that you could directly link comments :)